packages/ott-vis-panel/src/components/TreeDisplay.tsx

```diff
@@ -65,7 +65,7 @@ function buildMonolithTrees(monoliths: Monolith[]): TreeNode[] {
 				id: room.name,
 				region: monolith.region,
 				group: "room",
-				children: Array.from({ length: room.clients }, (_, index) => {
+				children: Array.from({ length: room.clients.length }, (_, index) => {
```

Comment:
this `Array.from` hack can be removed in favor of a `.map` or something more readable.